irrigation/jsonld-contexts/irrigation.jsonld

```diff
@@ -27,6 +27,8 @@
         "irrigationWeekday": "https://vocab.egm.io/irrigationWeekday",
         "irrigationDoseMinute": "https://vocab.egm.io/irrigationDoseMinute",
         "irrigationDailyDuration": "https://vocab.egm.io/irrigationDailyDuration",
+        "isWateringEnalbed": "https://vocab.egm.io/isWateringEnalbed",
+        "isWateringLimitationEnalbed": "https://vocab.egm.io/isWateringLimitationEnalbed",
```

Comment:
good but now, you have 2 `enalbed` 😆

Reply:
Yes, we may want to disable the watering (i.e. no irrigation, we block the valve) 
and on the other hand we may want to disable the watering limitation (i.e. no limitation is applied we don't touch the valve) 

The normal case is isWateringEnalbed = true & isWateringLimitationEnalbed = true i.e, there is some irrigation AND we apply the limitation recommmended by the algorithm

Is that somewhat clear ?